plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -284,6 +284,13 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("list", "watch").Groups(batchGroup).Resources("jobs", "cronjobs").RuleOrDie(),
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "system:kube-csr"},
+			Rules: []rbac.PolicyRule{
+				eventsRule(),
+				rbac.NewRule("get", "create", "list", "watch").Groups(certificatesGroup).Resources("certificatesigningrequests").RuleOrDie(),
```

Comment:
> node-bootstrapper is a pretty bad name for a permission that I hope will be used more broadley than node-bootstrapping.

I anticipate that there might be additional permissions in the future we'd want to add to the node bootstrapper role (like being able to get a node to see if there is already a record for itself, etc), so I'd rather give the bootstrapper role to the bootstrap credential for now.

> Can we change it to something like csrequestor

If we want an additional separate role that is limited in purpose to requesting a certificate, that could make sense as well

> bind it to all authenticated users or a csrquestor group?

I wouldn't expose cert requests to all authenticated users or a fixed group yet...